tasks/level-1/5.3.1.yml

```diff
@@ -3,6 +3,24 @@
 
 # 5.3.1 Ensure password creation requirements are configured
 
+- name: 5.3.1 - Ensure password creation requirements are honoured by PAM
+  pamd:
+    name: "{{ item }}"
+    type: password
+    control: requisite
+    module_path: pam_pwquality.so
+    module_arguments: 'try_first_pass
+        retry=3'
```

Comment:
Are these two lines correct? They look funky.

Reply:
they do look funky. They work as intended - I have verified by looking at the resulting config files. Similar format in 5.3.2.

Ansible docs say that parameter is a list. I will convert these lines to list format after the merge.